html-forms/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div>Email</div>
+    <input type="text" name="Email"/>
+  <div>Why is coding cool?</div>
+    <select name="coding">
+      <option value="select">Select an Option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+  <div>Favorite Language (choose one)</div>
+    <input type="radio" name="language" value="HTML" /> HTML
+    <input type="radio" name="language" value="CSS" /> CSS
+    <input type="radio" name="language" value="Javascript" /> Javascript
+  <div>Would you like to sign up for some more information?</div>
+    <input type="checkbox" name="signup" value="yes" /> Sign me up!
+  <div>Comments</div>
+    <textarea></textarea>
+  <div></div>
```

Comment:
We want to avoid using empty divs for a line break, it is better to wrap the elements inside a div.